applications/asset_tracker_v2/src/main.c

```diff
@@ -479,6 +481,7 @@ void main(void)
 		k_sleep(K_SECONDS(5));
 		sys_reboot(SYS_REBOOT_COLD);
 	} else {
+		module_set_state(MODULE_STATE_READY);
```

Comment:
It's a bit confusing that the module now has parallell states, with one maintained inside the module and then the CAF state. Is this the only way to do it when CAF is used?

Reply:
The initializations are hooked to module state event from main.
I don't think we can change this with configuration option as Kconfig options come as strings and cannot be used by preprocessor for variable names.

Reply:
Okay, let's leave it as is. This will be un-confused when app module moves to its own file, anyway.